src/image_manager/test/test_SkopeoDriver.cpp

```diff
@@ -242,7 +242,9 @@ TEST(SkopeoDriverTestGroup, acquireAuthFile) {
             CHECK_FALSE(boost::filesystem::exists(expectedAuthFilePath));
 
             auto driver = image_manager::SkopeoDriver{config};
-            driver.acquireAuthFile(config->authentication, config->imageReference);
+            auto randomExpectedAuthFilePath = driver.acquireAuthFile(config->authentication, config->imageReference);
+            CHECK_TRUE(expectedAuthFilePath != randomExpectedAuthFilePath);
```

Comment:
Writing  here but the following comments apply to all  cases under this test.

I think the check and reassignment between `expectedAuthFilePath`  and `randomExpectedAuthFilePath` is not necessary and can be simplified to a direct assignment to `authFilePath`.

On the other hand, I suggest to check that the auth file is created in the expected directories (in the case of this code scope, the XDG Runtime Dir). For example, something like:
```
CHECK(authFilePath.parent_path() == (xdgRuntimeDir / "sarus"))
```

Reply:
Done - I think I kept the check that the filename is unique, to make sure that I really do not use the old implementation. Anyway I removed it now, since I'm almost certain that it does not create the old static filename.